src/layout/FileUploadWithTag/EditWindowComponent.tsx

```diff
@@ -152,7 +153,7 @@ export function EditWindowComponent(props: EditWindowProps): JSX.Element {
               disabled={saveIsDisabled}
               className={classNames(classes.select, 'custom-select a-custom-select', {
                 'validation-error': props.attachmentValidations.filter((i) => i.id === props.attachment.id).length > 0,
-                'disabled !important': props.attachment.updating ? true : props.readOnly,
+                'disabled !important': props.attachment.updating ? true : readOnly,
```

Comment:
I think this expression could be simplified to simething like the suggestion?
```suggestion
                'disabled !important': props.attachment.updating || readOnly,
```

_Looks like I just cleaned my glasses. Since I look at small things like this..._